src/nist.gov/SP800-53/rev5/xml/NIST_SP-800-53_rev5_catalog.xml

```diff
@@ -132,7 +137,8 @@
                <p>Develop, document, and disseminate to  <insert type="param" id-ref="ac-1_prm_1"/>:</p>
                <part name="item" id="ac-1_smt.a.1">
                   <prop name="label" value="1."/>
-                  <p><insert type="param" id-ref="ac-01_odp.03"/>  access control policy that:</p>
+                  <p>
+                     <insert type="param" id-ref="ac-01_odp.03"/>  access control policy that:</p>
```

Comment:
```suggestion
                  <p><insert type="param" id-ref="ac-01_odp.03"/>  access control policy that:</p>
```

Hey @wendellpiez, can we work through whatever is causing the newline between `<p>` and `<param>` declarations with the XSLT implementation of the catalog/profile resolution processing?

There was a bug earlier in the sprint, usnistgov/oscal-content#122 to resolve usnistgov/oscal-content#121, because the YAML processor we use in Python will try to add YAML control characters to encode embedded newlines in certain blocks that is weirdly incorrect and something we cannot disable in that implementation. This YAML tool for Python is more secure and better maintained that the upstream PyYAML default. Can we discuss that?

If it is a big lift, should I just roll back the changes by hand before merge?

Reply:
OK I think we are good to go here and pushed up the most recent copy.

In internal pipeline code, changes were made in debugging in https://gitlab.nist.gov/gitlab/oscal/oscal-tools/-/commit/65726eb01bb5dfc10178b5a1227427bb051bf166. This comment is now obsolete, so closing it and marking PR ready for general review.